inlang/source-code/sdk2/src/project/loadProject.ts

```diff
@@ -38,6 +39,24 @@ export async function loadProject(args: {
 	 *
 	 */
 	preprocessPluginBeforeImport?: PreprocessPluginBeforeImportFunction;
+	/**
+	 * The id of the app that is using the SDK.
+	 *
+	 * The is used for telemetry purposes. To derive insights like
+	 * which app is using the SDK, how many projects are loaded, etc.
+	 *
+	 * The app id can be removed at any time in the future
+	 */
+	applicationId?: string;
+	/**
+	 * The version of the application that is using the SDK.
+	 *
+	 * This is used for telemetry purposes. To derive insights like
+	 * which version of the SDK is used, how many projects are loaded, etc.
+	 *
+	 * The application version can be removed at any time in the future.
+	 */
+	applicationVersion?: string;
```

Comment:
To workaround the posthog `appVersion` problems, i used `applicationVersion`

Reply:
The problem, when we change appID, all our insights will crash.